src/component/CreateMember.css

```diff
@@ -0,0 +1,24 @@
+.create-member {
+  margin: 0;
+  padding: 0;
+} 
+
+.create-member .create-button-member {
+  background-color: #078d14; /* Button background color */
+  padding: 10px;
+  margin: 20px;
+  color: #fff; 
+  border: none; 
+  cursor: pointer;
+  border-radius: 5px;
+  text-align: center;
+  font-size: 15px;
+  width: 90%;
```

Comment:
Using percentages for layout widths can lead to unpredictable behavior on different devices. Please make sure you are aware of it, or try to use flex or grid.